example/lib/main.dart

```diff
@@ -13,23 +14,19 @@ const simplePeriodicTask = "simplePeriodicTask";
 const simplePeriodic1HourTask = "simplePeriodic1HourTask";
 
 void callbackDispatcher() {
-  stderr.writeln('Debug - presumably performing a network request');
-
   Workmanager.defaultCallbackDispatcher((echoValue) {
-    stderr.writeln('Debug - Native echoed: $echoValue');
-
     switch (echoValue) {
       case simpleTaskKey:
-        print("$simpleTaskKey was executed");
+        stderr.writeln("$simpleTaskKey was executed");
```

Comment:
why?

Reply:
So that Flutter logs actually appear on Xcode's console when running the project from Xcode (useful for native debug purposes)